IS0401Test.py

```diff
@@ -320,8 +320,10 @@ def test_05(self):
                     return test.FAIL("First heartbeat occurred too long after initial Node registration.")
 
             # Ensure the heartbeat request body is empty
-            if heartbeat[1]["payload"] is not None:
-                return test.FAIL("Heartbeat POST contained a payload body.")
+            if heartbeat[1]["payload"] is not bytes():
+                return test.WARNING("Heartbeat POST contained a payload body.", "https://amwa-tv.github.io/nmos-discovery-registration/tags/v1.2.1/docs/2.2._APIs_-_Client_Side_Implementation_Notes.html#empty-request-bodies")

```

Comment:
@andrewbonney I don't quite like the fact that the 'More Information' link is to a specific version tag, that won't always be the latest one, nor even necessarily the spec version under test.

I wonder whether GenericTest __init__ could store the `spec_branch` into the `api` object, and we could pull that out here to form the link. Or is that a finesse too far?!

Reply:
I'm happy with that if you'd like to. I've got a similar issue all over the wiki where it only really makes sense to reference a particular version. There is of course the additional danger that the file doesn't exist in a particular branch.

It wouldn't totally solve this, but I'm wondering if the github.io docs could have an additional path for /latest/ or similar which always links to the most recent release spec.

Reply:
> I'm wondering if the github.io docs could have an additional path for /latest/ or similar

I wondered that too. That would basically make links to the spec kind of equivalent to what we have with the unversioned content on the wiki now.

But especially when the link refers to the spec, as a test suite user I might be happier if it linked to the version I was testing...

Happy with either, but I'll see if the branch-based link is easy.

Reply:
> There is of course the additional danger that the file doesn't exist in a particular branch.

That's just a fact, and affects any refactoring of the spec repos or wikis that we do.

>  I'll see if the branch-based link is easy.

It was, phew.